### DISCUSSION TITLE
Using the placemark/polyline package

### DISCUSSION THREAD

**Discussion by clementmas** [maplibre/maplibre-gl-directions#211] [Ideas]
There's a new polyline package from Placemark:
- https://github.com/placemark/polyline
- https://github.com/mapbox/polyline

As highlighted in the readme, it is maintained, it uses [lon, lat] order, it doesn't have any depencencies and it is typed.

Using it would simplify the decoding process from `decode(geometry).map((coordinate) => coordinate.reverse())` to `decode(geometry)`.

I'd like to use it in my own codebase but I don't want to use both since Mapbox's package is a dependency of maplibre-gl-directions.

So it'd be great if you would consider switching packages.

**Comment by smellyshovel** [comment]
@clementmas sounds reasonable enough to me.
I'm gonna become busy implementing the support for dynamic routing profiles soon, so a PR for this idea would be very welcome.

**Comment by smellyshovel** [comment]
Closing as complete by #214. Thanks, @clementmas, for the idea and the implementation!
